python_modules/dagster/dagster/_core/remote_representation/external.py

```diff
@@ -94,12 +94,12 @@ def __init__(
         self,
         repository_snap: RepositorySnap,
         repository_handle: RepositoryHandle,
-        instance: DagsterInstance,
+        auto_materialize_use_sensors: bool = True,
```

Comment:
this default is just useful for tests but I can remove it if it seems dangerous

Reply:
I think this default should probably be removed -- it does indeed feel scary to not force people to wire this up

Reply:
ok sounds good removed the default